Makefile

```diff
@@ -56,3 +56,24 @@ kafka-producer-run:
 				--bootstrap-server kafka:9092 \
 				--topic TestTopic \
 		"
+
+python-consume-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-dos-int-dev/deployment)"
+	python application/consume_message.py
+
+python-peek-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-dos-int-dev/deployment)"
+	python application/peek_message.py
+
+python-put-message-run:
+	eval "$$(make secret-fetch-and-export-variables NAME=uec-dos-int-dev/deployment)"
+	python application/put_message.py
+
+python-pytest:
+	python -m pytest application/tests -s -x -v
```

Comment:
Can we change that line to `make docker-run-tool CMD="python -m pytest ./tests -s -x -v" DIR=./application`